worker/src/Channel/Request.cpp

```diff
@@ -159,13 +178,15 @@ namespace Channel
 		if (reason != nullptr)
 			jsonResponse["reason"] = reason;
 
-		this->channel->Send(jsonResponse);
+		Request::channel->Send(jsonResponse);
 	}
 
 	void Request::TypeError(const char* reason)
 	{
 		MS_TRACE();
 
+		MS_ASSERT(Request::channel != nullptr, "channel unset");
+
```

Comment:
Let's replace it with `MS_ASSERT(Request::channel, "channel unset");` and let's remove the empty line between both `MS_ASSERT()`s.